flask/flask-launcher/src/main/java/net/corda/flask/launcher/Launcher.java

```diff
@@ -126,7 +130,10 @@ final int launch(Manifest manifest, List<String> jvmArgs, List<String> javaAgent
             if(jvmArgs != null) {
                 builder.getJvmArgs().addAll(jvmArgs);
             }
-            if(javaAgents != null) {
+            boolean disableJavaAgents = Optional.ofNullable(
+                    System.getProperty(Flask.JvmProperties.NO_JAVA_AGENT))
+                    .map(Boolean::parseBoolean).orElse(false);
```

Comment:
What about using [`Boolean.getBoolean(NO_JAVA_AGENT)`](https://docs.oracle.com/javase/8/docs/api/java/lang/Boolean.html#getBoolean-java.lang.String-)?

Reply:
I wasn't aware of its existence